cms/teams/models.py

```diff
@@ -0,0 +1,38 @@
+from functools import cached_property
+from typing import ClassVar
+
+from django.db import models
+from wagtail.admin.utils import get_user_display_name
+from wagtail.search import index
+
+
+class Team(index.Indexed, models.Model):
+    identifier = models.CharField(max_length=255, unique=True)
+    name = models.CharField(max_length=255)
+    precedence = models.PositiveIntegerField(null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    is_active = models.BooleanField(default=True)
```

Comment:
Question: what's the role of the `is_active` field? I can't find a reference to it in the attached cards. 
Also, wouldn't we want to make it changeable in the UI?

Reply:
Teams can be marked as inactive in Florence. @MebinAbraham can expand on this more

Reply:
Discussed and resolved with Kacper. If the user is `Enabled` in AWS Cognito, they are considered active.